src/physics/grid/PhysicsGridCalculator.i.hh

```diff
@@ -26,39 +40,40 @@ namespace celeritas
  * the lower xs value by E. If bin >= prime_energy_index, scale the result by
  * 1/E.
```

Comment:
Mostly -- except for the fact that the stored "value" grid is effectively discontinuous at the prime energy cutoff, so that for `bin == prime_index - 1` we have to scale the upper data by 1/E before interpolating 🙄 I can try to fix this in the current branch, but was planning on waiting to get the MR in for the other physics first.

Reply:
Ahh I see.. then never mind 🙂

Reply:
Actually it should be an easy fix. Let me take a whack at it.